python/ray/data/grouped_dataset.py

```diff
@@ -43,18 +46,79 @@ def __init__(self, dataset: Dataset[T], key: GroupKeyT):
         else:
             self._key = key
 
-    def aggregate(self, *aggs: AggregateFn) -> Dataset[U]:
-        """Implements an accumulator-based aggregation.
+    def agg(self, aggs: Union[Aggregation, List[Aggregation], Dict[
```

Comment:
```suggestion
    def agg(self, aggregate: Union[Aggregation, List[Aggregation], Dict[
```

For grouped dataset, can we just take `*aggs: Aggregation`?

Reply:
I think we need the same syntax sugar in both?

Reply:
Oh I see, then it should be just dict?

Reply:
Hmm weird I thought that I updated this, I'll fix it.